components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java

```diff
@@ -281,4 +285,31 @@ public static String getNotificationLocale() {
                 ? IdentityUtil.getProperty(I18nMgtConstants.NOTIFICATION_DEFAULT_LOCALE)
                 : I18nMgtConstants.DEFAULT_NOTIFICATION_LOCALE;
     }
+
+    public static byte[] getContentByteArray(NotificationTemplate notificationTemplate) {
```

Comment:
Shall we add the method comments

Reply:
Fixed with: https://github.com/wso2-extensions/identity-event-handler-notification/pull/280/commits/d39163fecfa8daae90fe3e3ebe3b70b805b84349